README.md

```diff
@@ -1,2 +1,121 @@
-# todolist
-Implementation of ES6 and Webpack by creating a simple ToDo List App
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- - Acknowledgements
+- [🔭 Future Features](#future-features)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [todoList] <a name="about-project"></a>
+
+**[todoList]** is the webpage representing a todoList. In these webpage you can add your tasks and remove a task once completed from the list.   
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">JAVASCRIPT</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[key_feature_1]** Used linters.yml to prevent linters to break the project
+- **[key_feature_2]** you can add your tasks
+- **[key_feature_3]** you can remove your tasks
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+vs code or other editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/sajibarafatsiddiqui/technothirst.git
```

Comment:
- [ ] You did a great job working on your project documentation, however, this is not the correct link for this project, please consider fixing it. If you're using the template from past projects please pay special attention to these details as it is very easy to miss some project-related information! 🤓 🔧

Reply:
Corrected. Thanks